Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -7,22 +7,27 @@ namespace BootCamp.Chapter
     class Lesson3
     {
         public static void Demo()
+        {
+            ProcessPersonBMI(); //Is this what you mean? Demo now calls a function to do the info gathering, which calls the functions to process the input.
+            Pause();
+            ProcessPersonBMI(); //Second person processing (Could be done by adding a second Demo function to the Main?)
+            Pause();
+        }
+        public static void ProcessPersonBMI()
         {
             //Get input from the user.
-            string fName = GetStringInput("Enter your first name: ");
-            string lName = GetStringInput("Enter your last name: ");
+            string firstName = GetStringInput("Enter your first name: ");
+            string lastName = GetStringInput("Enter your last name: ");
             int age = GetIntInput("Enter your age: ");
-            float height = GetFloatInput("Enter your height (in M): ");
+            float height = GetFloatInput("Enter your height (in M): "); //Changed from cm to M in order to work with tests.
             float weight = GetFloatInput("Enter your weight (in kg): ");
 
             //Print back information and follow up with BMI calculation.
-            Console.WriteLine("{0} {1} is {2} years old, weighs {3} kg and is {4} cm tall.", fName, lName, age, weight, height);
-            Console.WriteLine("{0}'s BMI is {1}.", fName, CalcBMI(height, weight));
-
-            //Halt program so output can be read.
-            Console.ReadKey();
+            Console.WriteLine($"{firstName} {lastName} is {age} years old, weighs {weight} kg and is {height} M tall.");
+            Console.WriteLine($"{firstName}'s BMI is " + CalcBMI(weight, height));
+            //Tried breaking these print lines into their own method block, however it would need 5 input parameters and would need to be
+            //called in this block to keep the variables in scope, so seemd to be a bit pointless. I imagine there's a better way to do it.
         }
-
         public static float CalcBMI(float weight, float height)
```

Comment:
Don't try to make it shortened, use full words unless it's a common knowledge. Also use PascalCase.
``CalculateBmi()`` is a better name.